src/main/java/seedu/address/model/eatery/Category.java

```diff
@@ -46,7 +46,10 @@ public static Category create(String name) {
         Category cat = categories.get(name);
 
         if (cat == null) {
-            return new Category(name);
+            Category newCat = new Category(name);
+            categories.put(newCat.getName(), newCat);
+            return newCat;
```

Comment:
again, try to use the full name since the other attribute classes already followed this convention

Reply:
isnt the only class that uses fullName variable the Name class...?? and for that class it makes sense bc ppl have first name and a surname

Reply:
haha i meant category instead of cat